platform/ui/src/components/downloadDialog/DownloadDialog.js

```diff
@@ -0,0 +1,235 @@
+            <div className="container">
+
+              <div className="row">
+                <div className="col">
+                  Please specify the dimensions, filename, and desired type for the output image.
+                </div>
+              </div>
+
+              <div className="file-info row">
+                <div className="file-name col9">
+                  <input
+                    type="text"
+                    placeholder="File Name"
+                    value={this.state.fileName}
+                    onChange={e => {
+                      this.setState({ fileName: e.currentTarget.value })
+                    }}
+                    className="input-ohif"
+                    id="file-name"
+                  />
+                </div>
+                <div className="file-type col3">
+                  <select
+                    className="select-ohif"
+                    value={this.state.fileType}
+                    onChange={e => {
+                        this.setState({ fileType: e.currentTarget.value })
+                      }
+                    }
+                  >
+                    {this.state.fileTypeOptions.map(({ key, value }) => {
+                      return (
+                        <option key={key} value={value}>{key}</option>
+                      );
+                    })}
+                  </select>
+                </div>
+              </div>
+
+              <div className="show-annotations">
+                <label htmlFor="show-annotations" className="form-check-label">
+                  Show Annotations
+                  <input
+                    id="show-annotations"
+                    type="checkbox"
+                    className="form-check-input"
+                    checked={this.state.showAnnotations}
+                    onChange={e => {
+                      this.setState({ showAnnotations: e.target.checked });
+
+                      this.props.setCacheReferences(
+                        this.state.previewElementRef,
+                        this.props.activeEnabledElement,
+                        this.props.forceRenderUpdate,
+                        e.target.checked
+                      );
+
+                      this.props.toggleAnnotations(e.target.checked, true);
+                    }}
+                  />
+                </label>
+              </div>
+
+              <div className="sizes row">
+                <div className="sizes_width col">
+                  <input
+                    type="number"
+                    placeholder="Width"
+                    value={this.state.width}
+                    onChange={e => {
+                      this.setState({ width: e.currentTarget.value });
+                      this.props.onResize(null, 'width', this.state.width);
+                    }}
+                    className="input-ohif"
+                  />
+                </div>
+                <div className="sizes_height col">
```

Comment:
What naming pattern are you following? I don't recognize `sizes_height`

Reply:
It´s BEM (http://getbem.com/). its is a block called `sizes` within an element called `height`, I forgot one underscore, sorry.  :)


It should be sizes__height